docs/permission/getting-started.md

```diff
@@ -3,3 +3,133 @@ id: getting-started
 title: Getting Started
 description: How to get started with the permission framework as an integrator
 ---
+
+If you prefer to watch a video instead, you can start with this video introduction:
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/EQr9tFClgG0" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+Backstage integrators control permissions by writing a policy. In general terms, a policy is simply an async function which receives a request to authorize a given permission for a user and (optional) resource, and returns a decision on whether to authorize that permission. Integrators can implement their own policies from scratch, or adopt reusable policies written by others.
```

Comment:
the wording "authorize a permission" chafes a bit - "authorize an action"? But maybe that goes against the vernacular we've set overall

Reply:
Changed to "authorize an action" as we've used that phrasing in a few other places.